src/components/project/project.resolver.ts

```diff
@@ -144,6 +155,56 @@ export class ProjectResolver {
     return await this.projectService.getRootDirectory(id, session);
   }
 
+  @ResolveField(() => SecuredLocation)
+  async primaryLocation(
+    @Parent() project: Project,
+    @Session() session: ISession
+  ): Promise<SecuredLocation> {
+    const { value: id, ...rest } = project.primaryLocation;
+    const value = id
+      ? await this.locationService.readOne(id, session)
+      : undefined;
+    return { value, ...rest };
+  }
+
+  @ResolveField(() => SecuredLocationList)
+  async nonPrimaryLocations(
```

Comment:
How about `otherLocations`? I'd rather not define a property as the opposite of another property.